SlackChatbot/utils/slack_utils.py

```diff
@@ -0,0 +1,27 @@
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+from utils.config import SLACK_TOKEN
+
+client = WebClient(token=SLACK_TOKEN)
+
+def send_message(channel, message):
+    try:
+        client.chat_postMessage(channel=channel, text=message)
+    except SlackApiError as e:
+        print("Error sending message:", e.response["error"])
+
+def process_event(event):
```

Comment:
Add docstrings to process_event and send_message functions to clarify what specific Slack events and messages these functions handle. Clear documentation helps future contributors